client/src/client/client_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	producer "../../../producer/src/producer"
+	producer "project_aurum/producer/src/producer"
 )
```

Comment:
I know we've been discussing this but I can't run the tests unless the import path is `../../../producer/src/producer`.